### PR TITLE
feat/immigration

### PR DESCRIPTION
**Pull request by leneherng**
#### `Immigration` Dashboard 
- add `countries` to i18n namespace
- use Flagpedia's API for country flags
- add flagcdn.com to allowed hostname in next config
- customisable `Fallback` for `ImageWithFallback`

EDIT: Removed `Section` changes

### CODE REVIEW

File: apps/app/components/Section/index.tsx
Comment (irfancoder) [inline apps/app/components/Section/index.tsx:1]: this component is messed up. check how it looks like in `dashboard index` and `home`

File: apps/app/dashboards/demography/immigration/index.tsx [hunk 1]
```diff
@@ -24,14 +106,345 @@ const Immigration: FunctionComponent<ImmigrationProps> = ({}) => {
...
+                                console.error(e);
+                              });
+                          }
+                        }}
+                      />
+                    </div>
+                  </div>
+                  <p className="text-dim text-center text-sm">{t("country_desc")}</p>
+                </div>
+              </div>
```
Comment (irfancoder) [inline apps/app/dashboards/demography/immigration/index.tsx:232]: This design is off from figma.

**Review by irfancoder — CHANGES_REQUESTED** [review]

File: apps/app/dashboards/demography/immigration/index.tsx (hunk 1 above)
Comment (leneherng) [reply apps/app/dashboards/demography/immigration/index.tsx]: I changed the design slightly because the search was in an awkward position

Comment (irfancoder) [reply apps/app/dashboards/demography/immigration/index.tsx]: Yep, noted in the group chat. tyvm

**Review by irfancoder — APPROVED** [review]

**Pull request merged**